.jshintignore

```diff
@@ -61,5 +61,3 @@ tv_apps/smart-system/bower_components/**
 tv_apps/tv-deck/bower_components/**
 tests/atoms/remote_date.js
 tests/atoms/screenshot.js
-tests/performance/getappname_atom.js
-tests/performance/exec-sync.js
```

Comment:
I don't understand why these two lines are here. Neither of these files pass jshint. Did you mean to delete or modify the files in this patch?  Or did you mean to put these .jshintignore changes in a different patch that does delete or modify the files?